mocks/marshalizerMock.go

```diff
@@ -0,0 +1,53 @@
+package mocks
+
+import (
+	"encoding/json"
+	"errors"
+)
+
+// ErrMockMarshalizer -
+var ErrMockMarshalizer = errors.New("MarshalizerMock generic error")
+
+// MarshalizerMock that will be used for testing
+type MarshalizerMock struct {
```

Comment:
This mock keeps being copied in every project (I also had to copy it in sovereign notifier).
We should really move it to core (not necessarily here in this PR)

Reply:
thanks for the heads up, actually there is already a mock implementation in core, refactored to use it from there